### PR TITLE
Mostly golint passing version

### PR DESCRIPTION
**Pull request by CMartinUdden**
I had to change some names to stop golint complaining. Also some things didn't need to be exported.

Otherwise just comments

### CODE REVIEW

**Comment by CMartinUdden** [comment]
also removed underscores now ...

**Comment by CMartinUdden** [comment]
I had left in a non used boolean from other work

File: allow/container.go [hunk 1]
```diff
@@ -129,7 +130,7 @@ func AllowContainerCreate(req authorization.Request, config *types.Config) *type
 		for _, b := range cc.HostConfig.Binds {
 			vol := strings.Split(b, ":")
 
-			if !AllowVolume(vol[0], config) {
```
Comment (juliengk) [inline allow/container.go]: Let's keep the name for now

File: allow/container.go [hunk 2]
```diff
@@ -225,7 +228,8 @@ func AllowVolume(vol string, config *types.Config) bool {
 	return false
 }
 
-func AllowMount(vol string) bool {
```
Comment (juliengk) [inline allow/container.go]: Let's keep the name for now

File: allow/image.go [hunk 3]
```diff
@@ -14,22 +14,24 @@ import (
...
 
-	if !AllowImage(params["fromImage"][0], config) {
+	if !Image(params["fromImage"][0], config) {
 		return &types.AllowResult{Allow: false, Msg: fmt.Sprintf("Image %s is not allowed to be pulled", params["fromImage"][0])}
 	}
 
 	return &types.AllowResult{Allow: true}
 }
 
-func AllowImage(img string, config *types.Config) bool {
```
Comment (juliengk) [inline allow/image.go]: Let's keep the name for now

**Pull request closed without merging**